CHANGELOG.rst

```diff
@@ -2,6 +2,11 @@
 Release Notes
 =============
 
+Unreleased
```

Comment:
I plan to release this soon after into PyPI so just bump the version number. I suppose 1.8.0 is a reasonable version.